start.go

```diff
@@ -202,8 +202,7 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 				"--state-dir", root,
 				"--listen", filepath.Join(namespace, "namespaced.sock"),
 			)
-			cmd = exec.Command("runv", args...)
-			cmd.Path = path
+			cmd = exec.Command(path, args...)
```

Comment:
@laijs @Crazykev  Acutally there is no need to do this.
Because of line164 "path, err := osext.Executable()",  the value of path is "/proc/self/exec", so it doesn't matter with runv is in PATH.
Although through my test it is OK if we don't inherit ENV from oci-d, shoudn't we cant find qemu?  it just gets abs through "cmd, err := exec.LookPath(QEMU_SYSTEM_EXE)" ?